config/config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "database" : {
+        "port": "27017",
+        "uri": "localhost",
+        "collection": "PokeData"
+    },
+    "twitter" : {
+        "consumer_key": "46yqEVwOOkf7dtN5Zfqxw4Nzu",
```

Comment:
@swathi-ssunder I was waiting for everyone's feedback, but I have made the commit with the required changes